umpf

```diff
@@ -1862,6 +1864,56 @@ do_show() {
 	cleanup
 }
 
+### namespace: tig ###
+
+tig_topic() {
+	local reply
+	find_branch_rev "${content}"
+	if [ -n "${reply}" ]; then
+		echo "${reply}" >> "${STATE}/refs"
+	fi
+
+	# possibly-existing local topic branch
+	if ${GIT} rev-parse --verify -q "${content}" >/dev/null 2>&1; then
+		echo "${content}" >> "${STATE}/refs"
+	fi
+
+	# possibly-existing remote-tracking topic branch
+	# (might be different from the remote chosen above)
+	if ${GIT} rev-parse --verify -q "${content}@{u}" >/dev/null 2>&1; then
+		echo "${content}@{u}" >> "${STATE}/refs"
+	fi
+}
+
+tig_release() {
+	echo "${content}" >> "${STATE}/refs"
+}
+
+tig_hashinfo() {
+	echo "${content}" >> "${STATE}/refs"
+}
+
+### command: tig ###
+
+do_tig () {
+	local refs base ranges
```

Comment:
`refs` is an array, so I think it should be declared as one here.
```
	local base
	local -a refs
````

Reply:
Thanks, I've changed it.